spring-auto-restdocs-core/src/main/java/capital/scalable/restdocs/jackson/FieldDocumentationVisitorContext.java

```diff
@@ -156,4 +166,13 @@ private Attribute optionalAttribute(Class<?> javaBaseClass, String javaFieldName
 
         return descriptions;
     }
+
+    private String resolveDeprecatedMessage(Class<?> javaBaseClass, String javaFieldName) {
+        Field field = ReflectionUtils.findField(javaBaseClass, javaFieldName);
+        boolean isDeprecated = field == null
+                ? false
+                : field.getAnnotation(Deprecated.class) != null;
```

Comment:
Could be written as:
`field != null && field.getAnnotation(Deprecated.class) != null`